### PR TITLE
Book component

### PR DESCRIPTION
**Pull request by Dorian30**
## Summary

- Made first react native component (Book)
- Set up project assets. 
- Set both Eslint & Prettier configs.
- Added babel aliases. 
- Deleted unnecessary files & set project's folder structure.

### iPhone X

image

### iPhone SE

image

### iPhone 7

image

### iPhone 8

image

### Pixel 3 XL

image

### Pixel 3 

image

### Pixel 2 

image

## Trello Card

https://trello.com/c/GKoQZe62/5-componente-libro

### CODE REVIEW

File: src/app/components/Book/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import styles from './styles';
+
+function Book() {
+  return (
+    <View style={styles.bookContainer}>
+      <Image style={styles.bookCover} source={BookCover} />
+      <View style={styles.bookInfo}>
+        <Text style={styles.bookTitle}>A little Bird Told Me</Text>
+        <Text style={styles.bookAuthor}>Timothy Cross</Text>
```
Comment (guidoprinc) [inline src/app/components/Book/index.js]: you could receive the image, title and author as props and then you can render different books with the same component

Comment (Dorian30) [reply src/app/components/Book/index.js]: On it! 🚀

File: src/app/screens/Library/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { View } from 'react-native';
+
+import Book from '@components/Book';
+import cover from '@assets/img_book1.png';
+
+function Library() {
+  return (
+    <View>
+      <Book cover={cover} title="A little bird Told Me" author="Timothy Cross" />
+    </View>
```
Comment (guidoprinc) [inline src/app/screens/Library/index.js]: when it's only a `<View>` in the container you can just use a Fragment like `<>...</>`

Comment (Dorian30) [reply src/app/screens/Library/index.js]: Awesome, didn't know you could use `fragment` with react native ?

**Pull request merged**